### DISCUSSION TITLE
css bootstrap 3 or 4 ?

### DISCUSSION THREAD

**Discussion by paskuale75** [simialbi/yii2-elfinder#9] [Q&A] [answered]
What version of bootstrap supported ?

**Comment by simialbi** [comment, marked as answer]
It's bootstrap 3 a.t.m.

**Comment by paskuale75** [comment]
![elfind-bootstrap](https://user-images.githubusercontent.com/9800/127678662-826b3591-0101-46fb-9e45-661997520b55.png)
there's a way to improve the style ?

**Comment by paskuale75** [comment]
![elfinder_file_preview](https://user-images.githubusercontent.com/9800/127679105-e614396d-f6f0-43d5-a710-7645fb75edcb.png)

**Comment by simialbi** [comment]
It's because JqueryUI CSS overrides styles:
inserting following in the config
```php
'components' => [
    'assetManager' => [
        'bundles' => [
            'yii\jui\JuiAsset' => [
                'css' => []
            ]
        ]
    ]
]
```
should do the trick

**Reply by paskuale75** [reply]
Great, now it work :rocket: !!!

**Reply by simialbi** [reply]
Pleased to hear
